core/test/stop/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 ginkgo_create_test(combined)
 ginkgo_create_test(iteration)
 ginkgo_create_test(time)
+ginkgo_create_test(stopping_status)
```

Comment:
alphabetical ordering - move it before test(time)

Reply:
Makes sense, is fixed now.